doc/integrator/fulltext_search.rst

```diff
@@ -0,0 +1,257 @@
+.. _integrator_fulltext_search:
+
+Full-text search
+================
+
+Enabling the *full-text search* feature involves adding a ``FullTextSearch``
+plugin to the viewer, and creating and populating a dedicated table in the
+database.
+
+The FullTextSearch plugin
+-------------------------
+
+The viewer should include a ``FullTextSearch`` plugin for the *text search*
+feature to be available in the user interface.
+
+See the `FullTextSearch API doc
+<http://docs.camptocamp.net/cgxp/2.3/lib/plugins/FullTextSearch.html>`_ for the
+list of options the plugin can receive.
+
+*The main viewer includes a FullTextSearch plugin by default.*
+
+The full-text search table
+--------------------------
+
+The *text search* feature requires a dedicated PostgreSQL table. The full-text
+search table should be named ``tsearch`` (for *text search*) and should be in
+the application-specific schema.
+
+You do not need to create the table yourself, as it is created by the
+``create_db`` command line (see the section
+:ref:`integrator_install_application`).
+
+If you did want to create the table manually you would use the following commands:
+
+.. prompt:: bash
+
+    sudo -u postgres psql -c "CREATE TABLE <schema_name>.tsearch (         id SERIAL PRIMARY KEY,         layer_name TEXT,         label TEXT,         public BOOLEAN DEFAULT 't',         params TEXT,         role_id INTEGER REFERENCES <schema_name>.role,         ts TSVECTOR);" <db_name>
```

Comment:
The request looks relay difficult to read, can you write it on more than one line?

Reply:
This was the generated file (from mako) and mako generation screwed up the formatting...
But I am removing the mako file for all files (except index.rst), so this problem disappears with the next commit.

Reply:
:-)